services/ui-src/src/components/forms/AdminBannerForm.test.tsx

```diff
@@ -5,48 +5,57 @@ import { axe } from "jest-axe";
 import { AdminBannerForm } from "components";
 import { bannerId } from "utils/constants/constants";
 import { convertDateEtToUtc } from "utils/time/time";
+import { errorHandler } from "utils/errors/errorHandler";
+
+jest.mock("utils/errors/errorHandler", () => ({
+  errorHandler: jest.fn(),
+}));
 
 const mockWriteAdminBanner = jest.fn();
+const mockWriteAdminBannerWithError = jest.fn(() => {
+  throw new Error();
+});
 
-const adminBannerFormComponent = (
+const adminBannerFormComponent = (writeAdminBanner: Function) => (
   <AdminBannerForm
-    writeAdminBanner={mockWriteAdminBanner}
+    writeAdminBanner={writeAdminBanner}
     data-testid="test-form"
   />
 );
 
+const fillOutForm = async (form: any) => {
+  // selectors for all the required fields
+  const titleInput = form.querySelector("[name='title']")!;
+  const descriptionInput = form.querySelector("[name='description']")!;
+  const startDateMonthInput = form.querySelector("[name='startDateMonth']")!;
+  const startDateDayInput = form.querySelector("[name='startDateDay']")!;
+  const startDateYearInput = form.querySelector("[name='startDateYear']")!;
+  const endDateMonthInput = form.querySelector("[name='endDateMonth']")!;
+  const endDateDayInput = form.querySelector("[name='endDateDay']")!;
+  const endDateYearInput = form.querySelector("[name='endDateYear']")!;
+  // fill out form fields
+  await userEvent.type(titleInput, "this is the title text");
+  await userEvent.type(descriptionInput, "this is the description text");
+  await userEvent.type(startDateMonthInput, "1");
+  await userEvent.type(startDateDayInput, "1");
+  await userEvent.type(startDateYearInput, "2022");
+  await userEvent.type(endDateMonthInput, "2");
+  await userEvent.type(endDateDayInput, "2");
+  await userEvent.type(endDateYearInput, "2022");
+  await userEvent.tab();
+};
```

Comment:
Moved this monster to its own external function because I'll be damned if I write it twice just to use it twice.